meson.build

```diff
@@ -190,6 +190,7 @@ if get_option('enable-dell')
 endif
 
 if get_option('enable-thunderbolt')
+  fwup = dependency('fwup', version : '>= 5')
```

Comment:
I think this is a mistake; I don't think thunderbolt depends on fwup at all. @gicmo ?